imap_processing/glows/l0/decom_glows.py

```diff
@@ -50,6 +50,11 @@ def decom_packets(
     histdata = []
     dedata = []
 
+    filename = packet_file_path.split("/")[-1]
+
+    # TODO: get this version from an offical source
```

Comment:
Do you want to store this in `glows/__init__.py` so you can `from imap_processing.glows import version` and get it globally for all the subsequent files and not have to pass it around in your function calls?